all_protos/cometbft/libs/bits/v1/types.proto

```
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package cometbft.libs.bits.v1;
```

Buf (1.54.0)

Files with package "cometbft.libs.bits.v1" must be within a directory "cometbft/libs/bits/v1" relative to root but were in directory "all_protos/cometbft/libs/bits/v1".

(PACKAGE_DIRECTORY_MATCH)